pkgs/applications/networking/browsers/tor-browser-bundle-bin/default.nix

```diff
@@ -379,7 +379,11 @@ stdenv.mkDerivation rec {
     cp $desktopItem/share/applications"/"* $out/share/applications
     sed -i $out/share/applications/torbrowser.desktop \
         -e "s,Exec=.*,Exec=$out/bin/tor-browser," \
-        -e "s,Icon=.*,Icon=web-browser,"
+        -e "s,Icon=.*,Icon=tor-browser,"
+    for i in 16 32 48 64 128; do
+      mkdir -p $out/share/icons/hicolor/''${i}x''${i}/apps/
+      ln -s $out/share/tor-browser/browser/chrome/icons/default/default$i.png $out/share/icons/hicolor/''${i}x''${i}/apps/tor-browser.png
```

Comment:
Why symlink is needed, can't you just/move instead?

Reply:
It could be used internally, which makes moving brittle. I don't see why a copy is better than a symlink.

Reply:
It is certainly possible to copy instead of symlinking. I chose symlinking because I saw many other packages doing the same.
Please let me know if this needs to be changed.

Reply:
Lots of packages where I've used similar code like this (like brave), use symlinks.
Not sure if it's used internally, but I think copying it to another place is wasteful.